project1/index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Cute+Font|Lato:300,300i,400,400i,700,700i,900,900i&display=swap"
+        rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css?family=Cute+Font|Lato:300,300i,400,400i,700,700i,900,900i|Roboto:300i,400,400i,500i,700,700i,900&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Prata&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <main>
+        <h1>
+            The Crane Wife <a href=https://longform.org/posts/the-crane-wife> </a> </h1> <h2>By
```

Comment:
A few things are going awry here on lines 21-24:

1. You should have a linebreak between your `h1` and `h2`
1. Your `cite` tag should be all lowercase
1. Your `a` element needs to be wrapped around the words "The Crane Wife"
1. The `href` value for your `a` element needs to be wrapped in double quotes (`"`)

This is what I would expect those lines to look like:
```html
        <h1><a href="https://longform.org/posts/the-crane-wife">The Crane Wife</a></h1>
        <h2>By <cite>CJ Hauser</cite></h2>
        <h3>July 16, 2019</h3>
```